src/component.py

```diff
@@ -117,27 +117,39 @@ def advanced_mode(self):
                 incremental = o.get('incremental', False)
                 primary_key = o.get('primary_key', [])
                 destination = o.get('kbc_destination')
+                legacy_manifest = o.get('legacy_manifest', False)
                 o = source
```

Comment:
Jak jsme se bavili na callu - Asi by bylo lepší vytáhnout ven, aby tam nebylo to o/source takový zmatený.